package.json

```diff
@@ -72,6 +72,7 @@
     "jquery": "3.5.0",
     "jschardet": "2.2.1",
     "keytar": "^5.5.0",
+    "mark.js": "^8.11.1",
```

Comment:
you will need to add this to https://github.com/microsoft/azuredatastudio/blob/main/remote/package.json and https://github.com/microsoft/azuredatastudio/blob/main/remote/web/package.json. You will also need to add the corresponding entries to https://github.com/microsoft/azuredatastudio/blob/main/src/vs/code/browser/workbench/workbench.html#L84 and https://github.com/microsoft/azuredatastudio/blob/main/src/vs/code/browser/workbench/workbench-dev.html#L84 also since the lib has a `.` in it will will need to special case it here https://github.com/microsoft/azuredatastudio/blob/main/src/vs/loader.js#L480

Also work with amir to make sure his patches scripts account for this.

I would also recommend running an adhoc build against this branch and ensure this work properly in web mode.

Reply:
Thanks. I'll be running another adhoc build with these changes.

@aaomidi can you comment on your patches scripts?

Reply:
Hmmmmm I'm not sure if a patch is going to be necessary since this does ship with a UMD. We're probably going to need to play around with the workbench.html file once this is done, but it probably won't need anything special.

Reply:
https://github.com/microsoft/azuredatastudio/blob/main/build/lib/rollup.ts The patch is here for angular, again I doubt we're going to need it for this one.

Reply:
adhoc build succeeded here after changes. thanks all!